templates/bundles/DemosPlanCoreBundle/DemosPlanCore/includes/base_footer.html.twig

```diff
@@ -47,12 +47,14 @@
                 </a>
             {% endif %}
 
+            {% block termsofuse %}
             {% if hasPermission('area_terms_of_use') %}
             <a class="c-footer__link" href="{{ path( 'DemosPlan_misccontent_static_terms') }}"
                aria-label="{{ 'terms.of.use'|trans }}">
                 {{- "terms.of.use"|trans -}}
             </a>
             {% endif %}
```

Comment:
Nitpick: Indentation

Reply:
done https://github.com/demos-europe/demosplan-core/pull/1083/commits/b8bcfd567dba92995d81ee0fc3c465fc15788a3d
i removed the twig block because solved with permission and just did code beautify like you suggested